{{cookiecutter.project_dashed}}-src/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# Docker Notes
+# ============
+# - `docker-compose up`: bring up containers
+# - `docker-compose up -d`: same, but run in background/daemon mode
+# - `docker-compose down`: bring down containers started with `-d`
+# - `docker ps`: show running containers
+# - `docker ps -a`: show all containers
+# - `docker-compose exec <container name> /bin/bash`: get shell in app container
+# - `docker images`
+# - `docker rmi <image name>`
+# - `docker stop $(docker ps -aq)`: stop all running containers
+# - `docker rm $(docker ps -a -q)`: remove all stopped containers
+
+version: '2'
+services:
+  {{ cookiecutter.project_dashed }}-pg:
+      image: postgres:9.6-alpine
```

Comment:
I don't think we are using PG-9.6 any more, can this be 10 or maybe 11?

Reply:
What's this "we" business? Of course we are using PG9.6. In at least one place, anyway. I'll take care of this.